OADP-4561: Update parent ToC for Troubleshooting modules (#89097)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/pods-crash-or-restart-due-to-lack-of-memory-or-cpu.adoc

:_mod-docs-content-type: ASSEMBLY
[id="pods-crash-or-restart-due-to-lack-of-memory-or-cpu"]
= Pods crash or restart due to lack of memory or CPU
include::_attributes/common-attributes.adoc[]
include::_attributes/attributes-openshift-dedicated.adoc[]
:context: pods-crash-or-restart-due-to-lack-of-memory-or-cpu
:namespace: openshift-adp
:local-product: OADP
:must-gather-v1-3: registry.redhat.io/oadp/oadp-mustgather-rhel9:v1.3
:must-gather-v1-4: registry.redhat.io/oadp/oadp-mustgather-rhel9:v1.4

toc::[]

If a Velero or Restic pod crashes due to a lack of memory or CPU, you can set specific resource requests for either of those resources.

The values for the resource request fields must follow the same format as Kubernetes resource requirements.
If you do not specify `configuration.velero.podConfig.resourceAllocations` or `configuration.restic.podConfig.resourceAllocations`, see the following default `resources` specification configuration for a Velero or Restic pod:

[source,yaml]
----
requests:
  cpu: 500m
  memory: 128Mi
----

[role="_additional-resources"]
.Additional resources

> * xref:../../backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc#oadp-velero-cpu-memory-requirements_about-installing-oadp[CPU and memory requirements]

Author: good catch, I've fixed it

:namespace: openshift-adp
:local-product: OADP
:must-gather-v1-3: registry.redhat.io/oadp/oadp-mustgather-rhel9:v1.3
:must-gather-v1-4: registry.redhat.io/oadp/oadp-mustgather-rhel9:v1.4

toc::[]

If a Velero or Restic pod crashes due to a lack of memory or CPU, you can set specific resource requests for either of those resources.

[role="_additional-resources"]
.Additional resources
* xref:../../backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc#oadp-velero-cpu-memory-requirements_about-installing-oadp[CPU and memory requirements]

include::modules/oadp-pod-crash-set-resource-request-velero.adoc[leveloffset=+1]
include::modules/oadp-pod-crash-set-resource-request-restic.adoc[leveloffset=+1]

[IMPORTANT]
====
The values for the resource request fields must follow the same format as Kubernetes resource requirements.
Also, if you do not specify `configuration.velero.podConfig.resourceAllocations` or `configuration.restic.podConfig.resourceAllocations`, the default `resources` specification for a Velero pod or a Restic pod is as follows:

[source,yaml]
----
requests:
  cpu: 500m
  memory: 128Mi
----
====

> 

Reviewer: I believe the handbook says that multiple points or ideas in a note should be conveyed as bullets. Maybe make your two notes two bullets. You might also want to consider pulling the code out and putting it in the docs. This is a really long note.

Author: For simplicity, removed this information from the admonition and added it before the include statement.

:must-gather-v1-4: registry.redhat.io/oadp/oadp-mustgather-rhel9:v1.4

toc::[]

If a Velero or Restic pod crashes due to a lack of memory or CPU, you can set specific resource requests for either of those resources.

The values for the resource request fields must follow the same format as Kubernetes resource requirements.
Also, if you do not specify `configuration.velero.podConfig.resourceAllocations` or `configuration.restic.podConfig.resourceAllocations`, the default `resources` specification for a Velero pod or a Restic pod is as follows:

[source,yaml]
----
requests:
  cpu: 500m
  memory: 128Mi
----

[role="_additional-resources"]
.Additional resources
* xref:../../backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc#oadp-velero-cpu-memory-requirements_about-installing-oadp[CPU and memory requirements]

include::modules/oadp-pod-crash-set-resource-request-velero.adoc[leveloffset=+1]
include::modules/oadp-pod-crash-set-resource-request-restic.adoc[leveloffset=+1]

[IMPORTANT]
====
The values for the resource request fields must follow the same format as Kubernetes resource requirements.

> Also, if you do not specify `configuration.velero.podConfig.resourceAllocations` or `configuration.restic.podConfig.resourceAllocations`, the default `resources` specification for a Velero pod or a Restic pod is as follows:

Reviewer 2: ```suggestion
If you do not specify `configuration.velero.podConfig.resourceAllocations` or `configuration.restic.podConfig.resourceAllocations`, see the following default `resources` specification configuration for a Velero or Restic pod:
```
